ui/pages/permissions-connect/snaps/snap-result/snap-result.js

```diff
@@ -140,17 +140,20 @@ export default function SnapResult({
       borderStyle={BorderStyle.none}
       flexDirection={FLEX_DIRECTION.COLUMN}
     >
+      <SnapAuthorshipHeader snapId={targetSubjectMetadata.origin} />
       <Box
-        className="snap-result__header"
+        className="snap-result__content"
         paddingLeft={4}
         paddingRight={4}
         alignItems={AlignItems.center}
         flexDirection={FLEX_DIRECTION.COLUMN}
+        style={{
+          overflowY: 'scroll',
```

Comment:
No need to show the scroll bar if the content isn't scrollable.
```suggestion
          overflowY: 'auto',
```